### PR TITLE
LPS-169697 Do not send User Notification that has as Recipient a role with no permission to VIEW the entry that triggers it

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-169697

### CODE REVIEW

**Comment by brianchandotcom** [comment]
@gabrielwas @p-albuquerque see 28ee54bfe72eadd05433013d9439427f1dd25ccc

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by gabrielwas** [comment]
LGTM

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx [hunk 1]
```diff
@@ -554,14 +547,12 @@ export default function EditNotificationTemplate({
...
 
-								{!Liferay.FeatureFlags['LPS-162133'] && (
-									<SingleSelect
-										disabled
-										label={Liferay.Language.get('type')}
-										options={[]}
-										value={Liferay.Language.get('email')}
-									/>
-								)}
+								<SingleSelect
```
Comment (naraandrade) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx:550]: @p-albuquerque you need to remove this SingleSelect. It was only meant to be displayed while the Feature Flag was disabled.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Pull request closed without merging**